### ISSUE TITLE
MarketplaceId.Id.1 parameter don't set dynamically 

### ISSUE THREAD

**Issue by alisanalacam** [keithbrink/amazon-mws-laravel#19]
I use AmazonOrderList class like this:

amazon-mws.php
```
'store1' => [
            'merchantId'       => '1',
            'marketplaceId'    => '1',
            'keyId'            => '1',
            'secretKey'        => '1',
            'mwsAuthToken'     => '1',
            'amazonServiceUrl' => 'https://mws-eu.amazonservices.com/',
            'muteLog'          => false,
        ],
```

----

```
$config = [
            'merchantId' => 'A**', 
            'marketplaceId' => 'A2***',
            'keyId' => 'AKIA****',
            'secretKey' => 'jl***',
            'mwsAuthToken' => 'amzn.mws.fcb***',
            'amazonServiceUrl' => 'https://mws.amazonservices.ae/',
        ];
$amz = new AmazonOrderList('store1');
$amz->setConfig($config);
...

```
MarketplaceId.Id.1 parameter is coming from config/amazon-mws.php file but it should change with my parameter which is set from me.

**Comment by keithbrink** [comment]
Yeah, there is definitely some oddness here, and I believe this is related to your other issue. I can't work on this today but hope to on the weekend. Feel free to ping me if you don't hear back by Monday.

**Comment by keithbrink** [comment]
Fixed in 7.0.2
